src/functions.php

```diff
@@ -356,11 +356,11 @@ function xml_adopt(SimpleXMLElement $to, SimpleXMLElement $from)
 /**
  * Upload cards to fritzbox
  *
- * @param string $xml
- * @param array $config
+ * @param SimpleXMLElement  $xmlNewPhonebook
+ * @param array             $config
  * @return void
  */
-function upload(string $xml, $config)
+function uploadPhonebook(SimpleXMLElement $xmlNewPhonebook, array $config)
```

Comment:
SimpleXMLElement $xmlPhonebook

It does not matter if its "new"

Reply:
OK